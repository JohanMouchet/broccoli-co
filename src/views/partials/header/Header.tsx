import React from "react";
import { ReactComponent as Logo } from "assets/images/logo.svg";
import { STATIC } from "constants/static";
import styles from "./Header.module.scss";

const Header: React.FC = () => {
  return (
    <header className={styles["header"]}>
      <span className={styles["logo"]}>
        <span className={styles["logo-icon"]}>
          <Logo />
        </span>
        <h2 className={styles["logo-text"]}>{STATIC.company.name}</h2>
      </span>
    </header>
  );
};

export default Header;
