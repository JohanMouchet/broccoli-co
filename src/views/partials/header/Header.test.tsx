import React from "react";
import { render } from "@testing-library/react";
import Header from "./Header";

it("Renders <Header /> without crashing", () => {
  expect(
    render(<Header>children</Header>).container.firstChild
  ).toMatchSnapshot();
});
