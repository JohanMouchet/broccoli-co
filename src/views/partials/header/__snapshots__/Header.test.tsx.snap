// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Header /> without crashing 1`] = `
<header
  class="header"
>
  <span
    class="logo"
  >
    <span
      class="logo-icon"
    >
      <svg>
        logo.svg
      </svg>
    </span>
    <h2
      class="logo-text"
    >
      Broccoli & Co.
    </h2>
  </span>
</header>
`;
